apiserver/api_server.go

```diff
@@ -253,6 +254,39 @@ func saveBitriseYMLFromJSONHandler(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, 200, SimpleResponse{Message: "OK"})
 }
 
+func loadSpecHandler(w http.ResponseWriter, r *http.Request) {
+	// Load steplib spec
+	cmd := command.New("bitrise", "stepman", "collections", "--format", "json")
+	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		respondWithErrorMessage(w, "Failed to get steplib spec, error: %s", err)
```

Comment:
shouldn't this log the output too? at least with `log.Println(out)` or something like that?